ui/.env

```diff
@@ -1,6 +1,6 @@
 PUBLIC_URL=.
 BROWSER=none
 REACT_APP_AUTH0_DOMAIN=sso.calyptia.com
-REACT_APP_AUTH0_CLIENT_ID=vsywuHQDbLFrz9TFHyLMXskZcSclHxEd
+REACT_APP_AUTH0_CLIENT_ID=hODtvC3oOQVwT8ILxoQFv5eRtgElsdFh
```

Comment:
I presume this is ok to commit, should it be a secret?

Reply:
It's not a secret. I can put it as a default instead and remove the .env file from git.

Reply:
Done.